controls/SV-204602.rb

```diff
@@ -32,6 +37,13 @@
     describe 'Control not applicable - SSH is not installed within containerized RHEL' do
       skip 'Control not applicable - SSH is not installed within containerized RHEL'
     end
+
```

Comment:
Not tested yet

Reply:
Tested it and it works but need to log an issue for SV-204588 that doesn't have the `7.4` release condition

Reply:
Added the missing condition for SV-204588 https://github.com/mitre/redhat-enterprise-linux-7-stig-baseline/pull/189/commits/17d8bc5f9ce482d6a0184a423ff55c9a84dc91c0